assets/style/main.css

```diff
@@ -0,0 +1,530 @@
+  height: 10px;
+  background-color: #c1c7d0;
+  border-radius: 50%;
+}
+
+.lang-link {
+  margin-right: 10px;
+  padding: 10px;
+  background-color: #ebebff;
+  border-radius: 8px;
+}
+
+.btn-more {
+  margin: 10px;
+  padding: 12px;
+  background-color: #fff;
+  border: 1px solid #396df2;
+  border-radius: 12px;
+  margin-left: 0;
+}
+
+.btn-more:hover {
+  color: #fff;
+  border: 1px solid #7f8cff;
+  background-color: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  border-radius: 8px;
+}
+
+.btn-more:disabled {
+  background-color: #fff;
+  border: 1px solid #5e6c84;
+  color: #5e6c84;
+}
+
+.btn-more:active {
+  background: #2230d2;
+  border-radius: 8px;
+  color: #fff;
+}
+
+.grid-item-3 {
+  border-top-right-radius: 50px;
+  margin: 30px;
+}
+
+.about-container {
+  display: flex;
+  align-items: flex-start;
+  margin: 30px;
+}
+
+.about-myself {
+  width: 45%;
+}
+
+.about-myexperience {
+  width: 55%;
+  margin: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.experience-list {
+  width: 90%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 20px;
+  margin-bottom: 10px;
+}
+
+.upward-icon {
+  rotate: 90deg;
+}
+
+.exper-lang {
+  display: flex;
+}
+
+.exper-lang-list {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.lang-img {
+  margin-bottom: 4px;
+  margin-right: 8px;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+}
+
+.list-hozin {
+  width: 92%;
+  border-bottom: 0.5px solid #dfe1e6;
+}
+
+.exper-link {
+  padding: 5px 10px;
+  background-color: #f7f7f9;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  border-radius: 8px;
+}
+
+.grid-item-4 {
+  min-width: 375px;
+  border-top-left-radius: 120px;
+  min-height: 100vh;
+  background-color: #6070ff;
+  background-image: url("../images/desktop/contact.png");
+  background-position: 0 0;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-bottom: 0;
+}
+
+.contact-container {
+  color: #fff;
+  width: 33%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.contact-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.contact-title {
+  color: #ebebff;
+  justify-content: center;
+  font-weight: 400px;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.input-bg {
+  width: 80%;
+  margin-top: 20px;
+  padding: 15px;
+  color: #172b4d;
+  background-color: #fff;
+  border: 1px solid #cfd8dc;
+}
+
+.footer {
+  position: relative;
+  width: 100%;
+  text-align: center;
+  display: none;
+}
+
+.end-page {
+  position: absolute;
+  height: 10px;
+  background-color: #000;
+  width: 25%;
+  border: 5px solid #000;
+  bottom: 10px;
+  left: 40%;
+}
+
+@media (max-width: 768px) {
```

Comment:
- [Optional] You are expected to build your portfolio mobile-first, so only desktop styling should be in a media query (@media (min-width: 768px)) and all mobile styling should be your default styling. Consider fixing this, please.